app/views/admin/portal_config/_metadata_tab.html.haml

```diff
@@ -0,0 +1,25 @@
+- if flash[:notice]
```

Comment:
where is this file called ?

Reply:
it called from this function 

https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/929/files#diff-b2b7b0a62c7bcca1112e0e0b7577d9a38a3ed1c191487ef4d3ca9526e7444a73R285
```ruby
def change_portal_config
    config_params = params.except(:controller, :action, :format, :authenticity_token, :commit)
    root_url = "#{rest_url}/"
    begin
      response = LinkedData::Client::HTTP.patch(root_url, JSON.parse(config_params.to_json))
      if response.success?
        flash[:notice] = 'Configuration updated successfully'
      else
        flash[:alert] = "Update failed: #{response.body}"
      end
    rescue StandardError => e
      flash[:error] = "An error occurred: #{e.message}"
    end

  end
  ```

Reply:
ah ok, you did the rendering in the helper function.
move the render to the controller action it is the normal way of doing in Rails.